### PR TITLE
implement operation name suffix ignore matcher

### LINKED ISSUE #63: add `ignore_suffix` configuration
It will work the same as java agent's. https://github.com/apache/skywalking/tree/master/docs/en/setup/service-agent/java-agent

### PR DESCRIPTION
**Pull request by Shikugawa**
Close #63.

It will behave the same as java agent's. If operation name on first span has ignore suffix, the segment won't be reported. And this configuration will be configurable via Skywalking CDS, then I will submit another PR to fix them.

### CODE REVIEW

**Comment by Shikugawa** [comment]
@wu-sheng PTAL

**Review by wu-sheng — COMMENTED** [review]
Please update the documentation about this config.

File: bazel/repositories.bzl [hunk 1]
```diff
@@ -81,3 +82,11 @@ def com_github_gabime_spdlog():
     build_file = "//bazel:spdlog.BUILD",
     urls = ["https://github.com/gabime/spdlog/archive/v1.7.0.tar.gz"]
   )
+
+def com_google_abseil():
+  http_archive(
+    name = "com_github_abseil-cpp",
```
Comment (wu-sheng) [inline bazel/repositories.bzl:88]: What does this repo for?

Comment (Shikugawa) [reply bazel/repositories.bzl:88]: This is common operation library used in Chromium (also in Envoy). I used this to introduce endsWith function. (Unfortunately, C++ STL doesn't have functionality that other languages have. So we should use these common libraries)

Comment (wu-sheng) [reply bazel/repositories.bzl:88]: Got it.

**Comment by Shikugawa** [comment]
@wu-sheng Generate API docs by protodoc. https://github.com/SkyAPM/cpp2sky/pull/64/files#diff-0b5ca119d2be595aa307d34512d9679e49186307ef94201e4b3dfa079aa89938

**Review by wu-sheng — APPROVED** [review]

**Pull request merged**